projects/systelab-components/src/lib/contextmenu/abstract-context-menu.component.ts

```diff
@@ -5,12 +5,16 @@ import { ContextMenuOption } from './context-menu-option';
 @Directive()
 export abstract class AbstractContextMenuComponent<T> extends AbstractContextComponent<T> {
 
-	@ViewChildren('childdropdownmenu') public childDropdownMenuElement: QueryList<ElementRef>;
+	@ViewChildren('childdropdownmenu0') public childDropdownMenuElement0: QueryList<ElementRef>;
```

Comment:
0? Root level?

Reply:
In fact it has not sense because I have changed the way the recursively is done, so I will remove the 0